lib/dbservice/constants/mappings.ex

```diff
@@ -348,11 +353,61 @@ defmodule Dbservice.Constants.Mappings do
     },
     "start_date" => %{
       db_field: "start_date",
-      required: ["student", "batch_movement"],
+      required: ["student", "batch_movement", "teacher_creation"],
       optional: [],
       type: :date
     },
 
+    # Teacher creation fields
+    "first_name" => %{
+      db_field: "first_name",
+      required: ["teacher_creation"],
+      optional: [],
+      type: :string
+    },
```

Comment:
What I was thinking is let's use `user_first_name` only for this and other user fields just to avoid redundancy. Will talk with ops to standardize the headers